src/_utils/convertToTitleCase.ts

```diff
@@ -0,0 +1,7 @@
+const convertToTitleCase = (str: string): string => {
+    return str.replace(/_/g, " ").replace(/\w\S*/g, function (txt) {
```

Comment:
What does this regex expression do?

Reply:
capitalizes all words in a string separated by a space. like from `tail whip` to `Tail Whip`